### DISCUSSION TITLE
[Feature]: New deployment method -> Git Dockerfile and Docker compose method

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3173] [Feature Requests]
### Description

At the moment we can select a git repo and choose if we want to deploy it via docker compose or a dockerfile. It would be nice if we could use a git repo and select Dockerfile and Docker Compose -> so coolify uses the docker compose file of my repo and in the compose file the specific Dockerfile also in my repo is used. We can view both of them in the coolify UI.

### Minimal Reproduction (if possible, example repository)

- [ ] Add a git (GitHub) deployment method that allows both custom Dockerfile and custom Docker compose

-> Related issue: https://github.com/coollabsio/coolify/issues/2552

### Version

-

**Comment by AKRking** [comment]
is it possible now?
